### DISCUSSION TITLE
Supported browsers

### DISCUSSION THREAD

**Discussion by ratara5** [sws2apps/organized-app#1377] [Q&A] [answered]
Hi. Thanks for this great app.
Which browsers and versions are supported by this application?
I am using Google Chrome Version 119.0.6045.123 (Official Build) (64-bit) on Ubuntu 20.04. I don't know if my problem is related to it:

* Aftter login, when I click on any item or element at https://localhost:4050/, I get back this error: 'Uncaught TypeError: OnClick is not a function'. 
* When I try to visit Network: http://172.23.0.1:4050/, I get back this message: 'You are using unsupported browser for the Congregation Program for Everyone app. Make sure that your browser is up to date, or try to use another browser'.

The same applies to the use of Mozilla Firefox 124.0.1 (64-bit).

I want to contribute to this project, I hope I can spend some time to learn.

**Comment by rhahao** [comment]
Hello @ratara5. Thanks for your question. All modern and up-to-date browsers should open the app without issues. Have you tried opening it using http://localhost:4050? Please note that using http://`local-ip-address` is not supported, unless being served by secure `https` connection.

**Reply by ratara5** [reply]
Thanks for your reply @rhahao.  
Sorry. Yes, I was trying to access http://localhost:4050. The problem is still the same as I describe in the question.

**Reply by rhahao** [reply]
Hum... if so, could you please send some screenshots that are displayed in the Console Tab of your browser when the error occurs?

**Comment by ratara5** [comment]
Of course. Here some screenshoots:

APP
![Screenshot from 2024-03-28 13-12-00](https://github.com/sws2apps/organized-app/assets/85402518/6593991d-636c-4b35-ab75-70cca9a0ac1e)

API
![Screenshot from 2024-03-28 13-14-08](https://github.com/sws2apps/organized-app/assets/85402518/d89a051d-33bb-43d9-b31c-668d14c0986e)
![Screenshot from 2024-03-28 13-15-23](https://github.com/sws2apps/organized-app/assets/85402518/56df2524-15a6-49af-9b19-4567f5d8b547)

BROWSER http://localhost:4050
![Screenshot from 2024-03-28 13-19-01](https://github.com/sws2apps/organized-app/assets/85402518/b1c734e1-5a3e-40df-b47b-2afe30d34059)

BROWSER http://172.23.0.1:4050/
![Screenshot from 2024-03-28 13-20-19](https://github.com/sws2apps/organized-app/assets/85402518/5bcc1917-afa7-4653-90b5-5dcaa9fc90bc)

I have noticed that Chrome asking me for an update. But, it does not update in any way possible.
Some simple websites made with vite do work at moment in Chrome on my system.
In any case, the Organized app does not work with Firefox either.

Thanks for your help.

**Reply by rhahao** [reply, marked as answer]
Oh so sorry @ratara5 😄 Actually, if you are already landed in the Dashboard page, it means it works. The problem is these card buttons do not yet have any actions attached to them. That’s why the error.

**Reply by ux-git** [reply]
Yes, we're still in active development phase of the next Organized app release. We're excited to publish it as soon as it's developed and finalised. @ratara5 if you have React skills and would like to help us, you are very welcome to do so! 😊🤝 Just let us know!
